src/main/containers/TestPanel.jsx

```diff
@@ -1,30 +1,70 @@
 import React, { useState } from 'react';
-import DataTree from '../components/DataTree.jsx';
 // sample JSON to pass down as props. Will be able to remove as the project evolves.
-import { largeData, smallData } from '../dummyData';
+// import { largeData, smallData } from '../dummyData';
+import DataTree from '../components/DataTree.jsx';
 import DataCanvas from './DataCanvas.jsx';
 import StyledPanel from './StyledPanel.jsx';
 
 // will need to get data from the get request to pass to the formatted view
 
 const TestPanel = (props) => {
-  const { data, updateTreeCount } = props;
-  const [mockups, setData] = useState([props.data]);
+  const { treeCount, updateTreeCount, data, setTests, tests } = props;
```

Comment:
Same as above—I think we need to split this into a few lines to keep the linters happy
```suggestion
  const { 
  treeCount, updateTreeCount, data, setTests, tests 
  } = props;
```